### PR TITLE
update python to 3.11.9

### PR DESCRIPTION
**Pull request by willkg**
- Update Python to 3.11.9
- Update fakesentry to Python 3.11.9 and switch to debian bullseye
- Fix interpreter in sentry-wrap.py

### CODE REVIEW

**Comment by willkg** [comment]
The CircleCI pipeline failed because it was still set up as a webhook. It's not required by branch protections. I removed the webhook, but the check status will be there until we update the PR. I'll do that next.

File: bin/sentry-wrap.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
-#!/usr/bin/python
+#!/usr/bin/env python
 
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 # Wraps a command such that if it fails, an error report is sent to the Sentry service
-# specified by SENTRY_DSN in the environment.
+# specified by ELIOT_SECRET_ELIOT_SECRET_SENTRY_DSN in the environment.
```
Comment (relud) [inline bin/sentry-wrap.py:8]: seems like a typo here:
```suggestion
# specified by ELIOT_SECRET_SENTRY_DSN in the environment.
```

**Review by relud — APPROVED** [review]
r+wc

File: bin/sentry-wrap.py (hunk 1 above)
Comment (willkg) [reply bin/sentry-wrap.py:8]: Boo search-and-replace!

**Comment by willkg** [comment]
Thank you!

**Pull request merged**